inference_segmentation.py

```diff
@@ -157,32 +156,24 @@ def segmentation(param,
     Returns:
 
     """
-    subdiv = 2
+    use_hanning = True
     sample = {"image": None, "mask": None, 'metadata': None}
     start_seg = time.time()
     print_log = True if logging.level == 20 else False  # 20 is INFO
-    pad = chunk_size * 2
-    h_padded, w_padded = [side + pad for side in input_image.shape]
-    dist_samples = int(round(chunk_size * (1 - 1.0 / 2.0)))
-
     model.eval()  # switch to evaluate mode
 
     # initialize test time augmentation
-    transforms = tta.Compose([tta.HorizontalFlip(), ])
+    transforms = tta.aliases.d4_transform()
```

Comment:
Could you include this change in the PR description as this seems like it could have a big impact and it's not directly related to Hann window implementation

Reply:
I understand this, for next time I would keep this in mind to keep things separated better. I added all the small tweaks to improve prediction performance however marginal.